java/maven/cloudbuild.yaml

```diff
@@ -0,0 +1,8 @@
+# In this directory, run the following command to build this builder.
+# $ gcloud container builds submit . --config=cloudbuild.yaml
+
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/maven', '.']
```

Comment:
Does that imply that you should be able to do something like `gcr.io/$PROJECT_ID/java/mvn package`? As written, this image would need to be run as `gcr.io/$PROJECT_ID/java/mvn mvn package`